github/github.go

```diff
@@ -68,6 +67,57 @@ type GithubConfig struct {
 	DryRun bool
 
 	useMemoryCache bool
+
+	analytics analytics
+}
+
+type analytic int
```

Comment:
why not use prometheus metrics (or any other metric type that we can consume w/ a server?)